src/r2mm/installing/profile_installers/GenericProfileInstaller.ts

```diff
@@ -286,6 +287,40 @@ export default class GenericProfileInstaller extends ProfileInstallerProvider {
         }
     }
 
+    private async installPackageZip(profile: Profile, rule: ManagedRule, installSources: string[], mod: ManifestV2) {
+        /*
+            This install method repackages the entire mod as-is and places it to the
+            destination route. Essentially the same as SUBDIR_NO_FLATTEN, but as a
+            zip instead of a directory. The zip name will be the mod ID.
+         */
+        const destDir = path.join(profile.getPathOfProfile(), rule.route);
+        await FileUtils.ensureDirectory(destDir);
+        const destination = path.join(destDir, `${mod.getName()}.ts.zip`);
```

Comment:
Might be getting confused but does this include the author name?

Think mod.getFullName() might do it if it doesn't?

Reply:
This is a manifest which seems to do
```ts
this.setName(mod.getFullName());
```
leading the `getName()` to return the fullName. In fact it doesn't have the `fullName` property at all.